src/types/tdigest.h

```diff
@@ -150,3 +152,93 @@ inline StatusOr<double> TDigestQuantile(TD&& td, double q) {
   diff /= (lc.weight / 2 + rc.weight / 2);
   return Lerp(lc.mean, rc.mean, diff);
 }
+
+inline void AssignRankForEqualInputs(const std::vector<size_t>& indices, double cumulative_weight,
```

Comment:
Would it be better that we only allow the `RevRank` algorithm implementation only allows unique inputs?